string marker_name
string menu_title
int32 state
